rust/agama-cli/doc/backend-for-testing.md

```diff
@@ -1,21 +1,21 @@
-# How to set up a backend for testing this CLI frontend
+# How to set up a backend for testing the CLI frontend
 
-I needed a testing instance of the D-Installer backend so that this
-command-line frontend has something to talk to.
+I needed a testing instance of the Agama backend so that the
+Rust command-line frontend has something to talk to.
 
 ## Summary
 
 1. Take the container used for continuous integration (CI) testing of the
    backend
-2. Give it a git checkout of the backend
+2. Give it a git checkout of this repo
 3. Install the backend within the container
 4. Copy the frontend binary into the container
 
 ## Considered Alternatives
 
 My first plan had a different finale, 4. Make the D-Bus service visible
 ouside the container, but I hit an issue with D-Bus authentication, hopefully
-solvable.
+solvable. (Update: `xdg-dbus-proxy` seems to work, ask mvidner about it)
```

Comment:
There are some specific mentions to Martin and Josef. Is that intentional?

Reply:
Martin: It's an idea for improvement that I think is out of scope for this PR. If you have already tried/considered cross-container D-Bus communication, maybe we can discuss it and write the result here, now or later

Reply:
Don't worry, I was mentioning it just in case they were leftover. Somehow I did not expect to have that annotations directly in the document.